### ISSUE TITLE
Wrong implementation

### ISSUE THREAD

**Issue by ilyes495** [fxia22/pointnet.pytorch#3]
I think this implementation of PointNet is not consistent with the network in the paper or the Tensorflow implementation. 
There are two transform steps in the original paper, and only one is implemented in PointNetfeat(), I will try to implement what I think is in the paper and pull a request, or correct me if I'm wrong.

**Comment by fxia22** [comment]
Thanks for pointing out. Yes, I omitted the second transformation steps in paper and the orthogonal constraint loss for both steps.

**Comment by ilyes495** [comment]
Great. thanks for the explanation
